packages/ui/src/FormGroupHeading.tsx

```diff
@@ -16,22 +24,23 @@ export const FormGroupHeading: React.FC<Props> = ({
 }) => {
   const theme = useTheme();
   return (
-    <Flex
-      align='center'
-      justify='start'
+    <Button
       mb={3}
-      bg={theme.colors.gray[100]}
+      color={theme.colors.white[100]}
```

Comment:
try apply the right button variant and colors so we fix A11y without changing visible UI. Something like `variant='ghost'`+ `bg={theme.colors.gray[100]}` + text color modification should work

Reply:
Side note: Open/Close on Follower isn't that graceful. Effect of having less fields?